sim/shaman/runes.go

```diff
@@ -48,6 +51,29 @@ func (shaman *Shaman) ApplyRunes() {
 	shaman.applySpiritOfTheAlpha()
 }
 
+func (shaman *Shaman) applyShoulderRuneEffect() {
+	if shaman.Equipment.Shoulders().Rune == int32(proto.ShamanRune_RuneNone) {
+		return
+	}
+
+	switch shaman.Equipment.Shoulders().Rune {
+	case int32(proto.ShamanRune_RuneShouldersVolcano):
+		shaman.applyT1Elemental4PBonus()
+	case int32(proto.ShamanRune_RuneShouldersRagingFlame):
+		shaman.applyT1Elemental4PBonus()
+	case int32(proto.ShamanRune_RuneShouldersRefined):
+		shaman.applyT1Enhancement4PBonus()
+	case int32(proto.ShamanRune_RuneShouldersChieftain):
+		shaman.applyT1Enhancement6PBonus()
+	case int32(proto.ShamanRune_RuneShouldersWindwalker):
+		shaman.applyT1Tank2PBonus()
+	case int32(proto.ShamanRune_RuneShouldersShieldMaster):
+		shaman.applyT1Tank4PBonus()
+	case int32(proto.ShamanRune_RuneShouldersTotemicProtector):
+		shaman.applyT1Tank6PBonus()
+	}
+}
```

Comment:
Should runes get a method inside of  `runes.go` with a switch statement. I ordered these by Tier #, then Spec, then # of pieces